docs/source/installation.rst

```diff
@@ -0,0 +1,93 @@
+
+.. _installation:
```

Comment:
I'd call the file sth like `completion.rst` (`installation` sounds like installing git-machete itself rather than its completion)